README.md

```diff
@@ -1784,6 +1784,17 @@ The following extractors use this feature:
 * `po_token`:  Proof of Origin (PO) Token(s) to use. Comma seperated list of PO Tokens in the format `CLIENT.CONTEXT+PO_TOKEN`, e.g. `youtube:po_token=web.gvs+XXX,web.player=XXX,web_safari.gvs+YYY`. Context can be either `gvs` (Google Video Server URLs) or `player` (Innertube player request)
 * `player_js_variant`: The player javascript variant to use for signature and nsig deciphering. The known variants are: `main`, `tce`, `tv`, `tv_es6`, `phone`, `tablet`. Only `main` is recommended as a possible workaround; the others are for debugging purposes. The default is to use what is prescribed by the site, and can be selected with `actual`
 
+##### PO Token settings
+* `po_token`:  Proof of Origin (PO) Token(s) to use. Comma seperated list of PO Tokens in the format `CLIENT.CONTEXT+PO_TOKEN`, e.g. `youtube:po_token=web.gvs+XXX,web.player=XXX,web_safari.gvs+YYY`. Context can be either `gvs` (Google Video Server URLs) or `player` (Innertube player request)
+* `pot_debug`: Enable debug logging for PO Token fetching. Either `true` or `false` (default `false`)
+* `fetch_pot`: Policy to use for fetching a PO Token from providers. `always` to always try fetch a PO Token regardless if the client requires one for the given context. `when_required` to only fetch a PO Token if the client requires one for the given context (default)
+
+###### youtubepot-memory
+* `max_size`: Max size of the PO Token LRU memory cache (default 25)
```

Comment:
What's maximum size measured in? This seems ambiguous

Reply:
Maximum items in the cache

Reply:
```suggestion
* `max_size`: Number of PO Tokens to save in LRU memory cache (default 25)
```

Do we even need to expose this, though?

Reply:
> He's right. It's confusing since cache is usually given in MB.
> 
> ```suggestion
> * `max_size`: Number of PO Tokens to save in LRU memory cache (default 25)
> ```
> 
> Do we even need to expose this, though?

Yeah thinking about it, we probably don't need to expose it, I don't see a need since it's a temp cache. Might be something we can add later if requested.

Reply:
> Shouldn't yt-dlp just cache one Po Token per client, since afaik a new Po Token will only be fetched if the old one is invalidated or a different client? Or are there other cases where a new Po Token will be fetched (such as a different IP address)?
> 
> EDIT: On a side note, perhaps `tokens_cached` would be a better name for that argument?

Not quite how PO Tokens work. You want the cache by what the token is bound to (which is not necessary by client). See the webpo cache spec which describes how they are cached for web clients.